### ISSUE TITLE
[MNG-3159] IT 95 is failing on the trunk and maven-2.0.x branch

### ISSUE THREAD

**Issue by jira-importer** [apache/maven#4779]
**[Brett Porter](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=brettporter)** opened **[MNG-3159](https://issues.apache.org/jira/browse/MNG-3159?redirect=false)** and commented

---

**Affects:** 2.0.8, 3.0-alpha-1

**Issue Links:**
- [MNG-3180](https://issues.apache.org/jira/browse/MNG-3180) IT0095 is failing
 (_**"is duplicated by"**_)

**Comment by jira-importer** [comment]
**[Brett Porter](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=brettporter)** commented

[INFO] [verifier:verify {execution: verify}]
[error] [Verifier] File [/Users/brett/scm/maven/core-integration-testing/core-integration-tests/target/test-classes/it0095/sub1/target/effective-pom.xml] does not match regexp [scp://host/path/parent/sub1]
[INFO] ------------------------------------------------------------------------
[ERROR] BUILD ERROR
[INFO] ------------------------------------------------------------------------
[INFO] There are test failures
[INFO] ------------------------------------------------------------------------
[INFO] Trace
org.apache.maven.lifecycle.LifecycleExecutionException: There are test failures
at org.apache.maven.lifecycle.DefaultLifecycleExecutor.executeGoals(DefaultLifecycleExecutor.java:564)
at org.apache.maven.lifecycle.DefaultLifecycleExecutor.executeGoalWithLifecycle(DefaultLifecycleExecutor.java:480)
at org.apache.maven.lifecycle.DefaultLifecycleExecutor.executeGoal(DefaultLifecycleExecutor.java:459)
at org.apache.maven.lifecycle.DefaultLifecycleExecutor.executeGoalAndHandleFailures(DefaultLifecycleExecutor.java:311)
at org.apache.maven.lifecycle.DefaultLifecycleExecutor.executeTaskSegments(DefaultLifecycleExecutor.java:278)
at org.apache.maven.lifecycle.DefaultLifecycleExecutor.execute(DefaultLifecycleExecutor.java:143)
at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:334)
at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:125)
at org.apache.maven.cli.MavenCli.main(MavenCli.java:280)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
at java.lang.reflect.Method.invoke(Method.java:585)
at org.codehaus.classworlds.Launcher.launchEnhanced(Launcher.java:315)
at org.codehaus.classworlds.Launcher.launch(Launcher.java:255)
at org.codehaus.classworlds.Launcher.mainWithExitCode(Launcher.java:430)
at org.codehaus.classworlds.Launcher.main(Launcher.java:375)
Caused by: org.apache.maven.plugin.MojoExecutionException: There are test failures
at org.apache.maven.plugin.verifier.VerifierMojo.execute(VerifierMojo.java:74)
at org.apache.maven.plugin.DefaultPluginManager.executeMojo(DefaultPluginManager.java:443)
at org.apache.maven.lifecycle.DefaultLifecycleExecutor.executeGoals(DefaultLifecycleExecutor.java:539)
... 16 more

**Comment by jira-importer** [comment]
**[John Dennis Casey](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=jdcasey)** commented

The problem was actually in what it0095 was testing. It was expecting the sibling location of the module to affect the calculation of the DistributionManagement Site URL, which it never did. I changed it0095 to actually test all fields that are affected by a childPathAdjustment, added in MNG-2006 (project/scm/connection, project/scm/developerConnection, project/scm/url, and project/url).

I added a new integration test (it00128) which checks for the fix given in MNG-3134, i.e. the project/distributionManagement/site/url.
